ScareCrowController/ScareCrowController.pde

```diff
@@ -18,7 +18,7 @@ char decrease_keys[][] = { {'R',  'T',  'Y'}, {'U',  'I',  'O'}, {'P',  '{',  '}
 char lookup_controls[] = {'p', 'r', 'y'}; /* pitch, roll, yaw */
 char lookup_pid[]      = {'p', 'i', 'd'}; /* proportional, integral, derivative */
 float values[][]       = { {0, 0, 0}, {0, 0, 0}, {0, 0, 0} }; /* pitch, roll, yaw PIDs */
-float throttle         = 0;
+float throttle         = -180; /* For safety, so that propellors aren't triggered on startup if the drone is placed inclined*/
```

Comment:
This should be handled in the PID, not here. A -180 throttle does not guarantee that the output will be 0. 

@KarthikNayak Since you're redoing the PID, could you take a look at this? It should be handled where the output is sent to the motors.